test/on_yubikey/cli_piv/test_misc.py

```diff
@@ -27,4 +29,19 @@ def test_write_read_object(self):
             output = ykman_cli('piv', 'read-object', '0x5f0001')
             self.assertEqual('test data\n', output)
 
+        def test_export_invalid_certificate_fails(self):
+            ykman_cli('piv', 'write-object', hex(OBJ.AUTHENTICATION), '-',
+                      '-m', DEFAULT_MANAGEMENT_KEY,
+                      input='Kom ihåg att du aldrig får snyta dig i mattan!')
+
+            with self.assertRaises(SystemExit):
+                ykman_cli('piv', 'export-certificate',
+                          hex(OBJ.AUTHENTICATION), '-')
+
+        def test_info_with_invalid_certificate_does_not_crash(self):
+            ykman_cli('piv', 'write-object', hex(OBJ.AUTHENTICATION), '-',
+                      '-m', DEFAULT_MANAGEMENT_KEY,
+                      input='Kom ihåg att du aldrig får snyta dig i mattan!')
```

Comment:
Please use a more boring phrase, to avoid distraction. :)

Reply:
:laughing: